pre-course1,exercise-2.java

```diff
@@ -0,0 +1,56 @@
+// Time Complexity : push is 0(1),pop is o(n) cause for last element we need to iterate all the elements.
+// Space Complexity :12+12 for three integers variable.
+// Did this code successfully run on Leetcode :no,getting o is popped
+// Any problem you faced while coding this :yes regarding push or pop getting 0 is popped.
+
```

Comment:
Pushing - Add elements at the head of the LL
Pop - save the value at the head and move the head to head.next